mm2src/coins/eth.rs

```diff
@@ -5503,16 +5676,51 @@ pub async fn eth_coin_from_conf_and_request(
 
     let mut web3_instances = vec![];
     let event_handlers = rpc_event_handlers_for_eth_transport(ctx, ticker.to_string());
-    for node in nodes.iter() {
-        let transport = Web3Transport::new_http(vec![node.clone()], event_handlers.clone());
+    for url in urls.iter() {
+        let uri: Uri = try_s!(url.parse());
+
+        let transport = match uri.scheme_str() {
+            Some("ws") | Some("wss") => {
+                let node = WebsocketTransportNode {
+                    uri: uri.clone(),
+                    gui_auth: false,
+                };
+                let websocket_transport = WebsocketTransport::with_event_handlers(node, event_handlers.clone());
+
+                // Temporarily start the connection loop (we close the connection once we have the client version below).
+                // Ideally, it would be much better to not do this workaround, which requires a lot of refactoring or
+                // dropping websocket support on parity nodes.
```

Comment:
I don't think we should support parity nodes anymore, please correct me if I am wrong @artemii235.

Reply:
I think we should drop it's support as it's deprecated, but didn't want to make that decision on my own.

Reply:
I think we agreed to stop support for parity but I think the `client_version`/`web3_clientVersion` call should be kept   since it checks if the node is reachable and working on activation, and excludes it from the list if not. The right approach is to return an error on activation if one node is not reachable but I guess this can cause a problem for GUIs since if a public node that we don't control is down, coin activations won't work until the list/config is changed which requires a new GUI release. I will resolve this comment since we will keep the current approach.